src/training/flava_eval/finetune_glue.py

```diff
@@ -163,29 +202,39 @@ def get_task_metric(task_name):
 def get_task_dataloaders(args):
     tokenizer = get_tokenizer(args.model)
     task_name = args.task_name
+    roberta_tokenizer = isinstance(tokenizer, transformers.models.roberta.tokenization_roberta.RobertaTokenizer) or isinstance(tokenizer, transformers.models.roberta.tokenization_roberta_fast.RobertaTokenizerFast)
```

Comment:
I don't think this is quite right since OpenCLIP uses a wrapper class (you can see this in `tokenizer.py`) around the huggingface tokenizers. You can test your implementation by setting `--model= roberta-ViT-B-32` (and leaving `--pretrained` blank)

Reply:
yeah i have to do tokenizer.tokenizer